src/CommitmentsV2/SFA.DAS.CommitmentsV2.UnitTests/Application/Queries/GetApprenticeships/GetApprenticeshipsHandlerTests.cs

```diff
@@ -237,6 +243,91 @@ public class GetApprenticeshipsHandlerTests
             Assert.AreEqual("Should_Be_Second", actual.Apprenticeships.ElementAt(1).LastName);
         }
 
+
+        [Test, MoqAutoData]
+        public async Task And_No_Sort_Term_And_Is_And_There_Are_ApprenticeshipUpdates_These_Appear_First(
+            GetApprenticeshipsRequest request,
+            Mock<IAlertsMapper> alertsMapper,
+            [Frozen] Mock<ICommitmentsReadOnlyDbContext> mockContext)
+        {
+            //Arrange
+            request.SortField = null;
+            request.PageNumber = 0;
+            request.PageItemCount = 0;
+            request.ReverseSort = false;
+
+            var mapper = new ApprenticeshipToApprenticeshipDetailsMapper(alertsMapper.Object);
+            var apprenticeships = new List<Apprenticeship>
+            {
+                new Apprenticeship
+                {
+                    FirstName = "XX",
+                    LastName = "Should_Be_Second",
```

Comment:
Not sure if the last names should be changed to stop any confusion with the ordering and them. Its very minor but might be worth it.